variantlib/validators.py

```diff
@@ -184,9 +185,18 @@ def validate_variants_json(data: dict) -> None:
             raise ValidationError(
                 f"Invalid `variant_hash` type: `{variant_hash}` in data (expected str)"
             )
+
         if VALIDATION_VARIANT_HASH_REGEX.fullmatch(variant_hash) is None:
             raise ValidationError(f"Invalid variant hash `{variant_hash}` in data")
 
+        if variant_hash == "0" * VARIANT_HASH_LEN:
```

Comment:
Don't we need to check the converse now too?

Reply:
I'm not sure I understand what you said.

Reply:
We're checking that null variant has no properties. Shouldn't we check that regular variant has at least one property?

Reply:
Ah that makes sense.
I used to rely on the fact that VDesc validator would take care of that.
But for obvious reasons... Not anymore 😂

Reply:
Done